app/views/groups/group_links/_invited_groups.html.erb

```diff
@@ -1,38 +1,35 @@
-<%= turbo_frame_tag('members') do %>
-  <div class="flex flex-col">
-    <div class="overflow-x-auto">
-      <div class="shadow">
-        <table class="min-w-full table-fixed dark:divide-gray-600">
-          <thead class="text-gray-700 bg-gray-50 dark:bg-gray-700 dark:text-gray-400">
-            <tr>
-              <th scope="col" class="px-6 py-3 text-left"><%= t(:"groups.group_links.index.table_header.group") %></th>
-              <th scope="col" class="px-6 py-3 text-left"><%= t(:"groups.group_links.index.table_header.source") %></th>
-              <th scope="col" class="px-6 py-3 text-left"><%= t(:"groups.group_links.index.table_header.access_granted") %></th>
-              <th scope="col" class="px-6 py-3 text-left"><%= t(:"groups.group_links.index.table_header.access_level") %></th>
-              <th scope="col" class="px-6 py-3 text-left"><%= t(:"groups.group_links.index.table_header.expiration") %></th>
-              <th scope="col" class="px-6 py-3 text-right"><%= t(:"groups.group_links.index.table_header.action") %></th>
-            </tr>
-          </thead>
-          <tbody
-            id="invited-groups-table-body"
-            class="
-              bg-white
-              divide-y
-              divide-gray-200
-              dark:bg-gray-800
-              dark:divide-gray-700
-            "
-          >
-            <% @namespace_group_links.each do |namespace_group_link| %>
-              <%= render partial: "namespace_group_link",
-              locals: {
-                namespace_group_link: namespace_group_link
-              } %>
-            <% end %>
-          </tbody>
-        </table>
-      </div>
+<div class="flex flex-col">
```

Comment:
**nitpick (blocking):** Can you remove the 3 surrounding divs so that we match the `_member_listing.html.erb` partial.

Reply:
Removed in 83d659d1a9d8b48d234105e036d83a2ca67206dd